packages/core/src/codewhisperer/service/transformByQ/transformApiHandler.ts

```diff
@@ -640,6 +640,13 @@ export async function getTransformationPlan(jobId: string) {
         const arrowIcon = getTransformationIcon('upArrow')
 
         let plan = `<style>table {border: 1px solid #424750;}</style>\n\n<a id="top"></a><br><p style="font-size: 24px;"><img src="${logoIcon}" style="margin-right: 15px; vertical-align: middle;"></img><b>${CodeWhispererConstants.planTitle}</b></p><br>`
+        const authType = await getAuthType()
+        const linesOfCode = jobStatistics.find(
+            (stat: { name: string; value: number }) => stat.name === 'linesOfCode'
+        ).value
+        if (authType === 'iamIdentityCenter' && linesOfCode > 100000) {
+            plan += `<p>${linesOfCode} lines of code submitted for transformation, maximum charge of this transformation is $${(linesOfCode * 0.003).toFixed(2)} (this charge applies only after the free limit in your organization's subscriptions is exhausted). To prevent the charge, you can stop the job before the transformation completes.</p>`
```

Comment:
Can we pull this price calculation out into a utility function so its more obvious? It would also make it easier to update and keep track of later.
Is there a risk with the price updating and this calculation being stale, which would show  the user the wrong price?

Reply:
Good point about the price updating! We decided to link to our docs instead, will update PR.